packages/assets-controllers/src/TokensController.test.ts

```diff
@@ -334,9 +347,51 @@ describe('TokensController', () => {
     stub.restore();
   });
 
+  it('should add token by networkClientId', async () => {
```

Comment:
This makes it sound like it will be keyed by `networkClientId` rather than chainId. Maybe `should add token to the correct chainId when passed a networkClientId` or something like that?

Reply:
your suggestion definitely reads better